build/ci/github-actions/examples.md

```diff
@@ -919,10 +919,10 @@ jobs:
           tags: user/app:latest
       -
         name: Update repo description
-        uses: peter-evans/dockerhub-description@v2
+        uses: peter-evans/dockerhub-description@v3
         with:
-          username: ${{ secrets.DOCKERHUB_USERNAME }}
-          password: ${{ secrets.DOCKERHUB_PASSWORD }}
+          username: ${{ secrets.DOCKER_HUB_USERNAME }}
+          password: ${{ secrets.DOCKER_HUB_ACCESS_TOKEN }}
```

Comment:
This one is on purpose as this action requires plain password.

```suggestion
          password: ${{ secrets.DOCKER_HUB_PASSWORD }}
```

Reply:
Since this is different, probably worth adding a comment to indicate why :+1:

Reply:
In the docs is mentioned that a PAT or a password is required: 

> Docker Hub password or [Personal Access Token](https://docs.docker.com/docker-hub/access-tokens/) with `read/write/delete scope`.

https://github.com/peter-evans/dockerhub-description#action-inputs

I managed to run the action successfully with a PAT.

Reply:
Ohh didn't know we could use a PAT now! Nice